src/ReaderPluginManager.php

```diff
@@ -26,25 +29,9 @@ class ReaderPluginManager extends AbstractPluginManager
         'javaproperties'  => 'Zend\Config\Reader\JavaProperties',
     ];
 
-    /**
-     * Validate the plugin
-     * Checks that the reader loaded is an instance of Reader\ReaderInterface.
-     *
-     * @param  Reader\ReaderInterface $plugin
-     * @return void
-     * @throws Exception\InvalidArgumentException if invalid
-     */
-    public function validatePlugin($plugin)
```

Comment:
`validatePlugin()` is no longer necessary, as the `AbstractPluginManager` has an implementation that refers to the `$instanceOf` property for the most common use case of validating against a type.